src/engraving/rendering/dev/chordlayout.cpp

```diff
@@ -1823,6 +1823,34 @@ void ChordLayout::layoutChords1(LayoutContext& ctx, Segment* segment, staff_idx_
                 }
             }
 
+            // Create space between ledger lines and stems
+            // Above stave
+            Note* topUpNote = upStemNotes.back();
+            Note* bottomDownNote = downStemNotes.front();
+            if (!bottomUpNote->chord()->noStem() && bottomUpNote->chord()->stem()) {
```

Comment:
Is there a particular reason to check `noStem()`? As far as I know, whenever `noStem` is true, the stem is (supposed to be) removed, based on the value of `shouldHaveStem()`.